src/IPatchworkProtocol.sol

```diff
@@ -715,11 +754,19 @@ interface IPatchworkProtocol {
     function mintBatch(address to, address mintable, bytes calldata data, uint256 quantity) external payable returns (uint256[] memory tokenIds);
 
     /**
-    @notice Set the protocol fee configuration
+    @notice Proposes a protocol fee configuration
     @dev must be protocol owner or banker to call
+    @dev configuration does not apply until commitProtocolFeeConfig is called
     @param config The protocol fee configuration to be set
     */
-    function setProtocolFeeConfig(ProtocolFeeConfig memory config) external;
+    function proposeProtocolFeeConfig(ProtocolFeeConfig memory config) external;
```

Comment:
Is it worth adding getters for proposals? Or just have people rely on logs for that?

Reply:
I was thinking logs are fine